### DISCUSSION TITLE
Request access to DLR tool Modgen or documentation "Klimmek, Thomas. ModGen User’s Manual. Version 2023-01-17. 2023"

### DISCUSSION THREAD

**Discussion by Preethamkadambi** [DLR-AE/LoadsKernel#49] [Questions & Answers]
Hello, 
I was going through "Set-up of a generic, aeroelastic simulation model for demonstration and software testing purposes Master Thesis"  by Francisco Carvalho and "PRELIMINARY STRUCTURAL DESIGN OF A HIGH ASPECT RATIO TRANSPORT AIRCRAFT WITH LAMINAR WING" by Markus R. Ritter et al. There are multiple citations of the CPCAS Mona and "MODGEN" and the tool user guide. The entire process of setting of the mass model, aerodynamics and structural model from the aircraft specification/datasheet is rather intensive. In the spirit of learning and comparing the results by the end user. I would like to request access to this tool.

**Comment by ArneVoss** [comment]
Hello Preetham,

please note that not all our tools are published. In most cases, the modeling can be done manually as well and 
 having access to a bunch of tools doesn't ensure good and reliable results.
If you want to learn more about aeroelastic modelling, please look for the scientific publications by the authors Klimmek and Schulze, there are a number of articles where their software is used. Other authors that are worth a look include Handojo, Dillinger or myself. A good starting point for your literature study is elib.dlr.de or simply scholar.google.com. For newer publications, we usually also upload the PDF.

I hope this helps, 
Arne
